ci/docker/android-builder/Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 ENV ANDROID_HOME /opt/android-sdk
 ENV PATH ${PATH}:${ANDROID_HOME}/tools/:${ANDROID_HOME}/platform-tools:${ANDROID_HOME}/platform-tools/bin:${ANDROID_HOME}/tools/bin
 # https://developer.android.com/studio/index.html#command-tools
-ARG ANDROID_SDK_URL=https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
+ARG ANDROID_SDK_URL=https://dl.google.com/android/repository/commandlinetools-linux-6609375_latest.zip
```

Comment:
It's been cherry-picked from @int02h 's changes for Java 11.

Reply:
Tell more about motivation do this, please

Reply:
The old version of the command line tools is incompatible with Java 11.
This PR is a partial backport from changes for Java 11. 
Daniil will update an image in another PR.